docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1249 @@
+      </sect3>
+    </sect2>
+
+    <sect2>
+      <title>The Security Values</title>
+
+      <para>This section will look at the values.yaml sections dealing with
+      the system security components.</para>
+
+      <sect3>
+        <title>Certificates</title>
+
+        <para>The certificates section can be used to enable the cert-manager
+        to generate TLS certificates for each component in the HPCC Systems
+        deployment.</para>
+
+        <programlisting>certificates:
+  enabled: false
+  issuers:
+    local:
+      name: hpcc-local-issuer</programlisting>
+
+        <para>In the delivered yaml file certificates are not enabled, as
+        illustrated above. You must first install the cert-manager to use this
+        feature.</para>
+      </sect3>
+
+      <sect3 id="ValYAML_Secrets">
+        <title>Secrets</title>
+
+        <para>The Secrets section contains a set of categories, each of which
+        contain a list of secrets. The Secrets section is where to get info
+        into the system if you don't want it in the source. Such as code with
+        embedded code, you can have that defined in the code sign sections. If
+        you have information that you don't want public but need to run it you
+        could use secrets.</para>
+      </sect3>
+
+      <sect3>
+        <title>Vaults</title>
+
+        <para>Vaults is another way to do Secrets. The vaults section mirrors
+        the secret section but leverages <emphasis>HashiCorp Vault</emphasis>
+        for the storage of secrets. There is an additional category for vaults
+        named "ecl-user". The intent of the ecl-user vault secrets is to be
+        readable directly from ECL code. Other secret categories are read
+        internally by system components and not exposed directly to ECL
+        code.</para>
+      </sect3>
+    </sect2>
+  </sect1>
+
+  <sect1 id="TheValuesYaml_FileAndRelated" role="nobrk">
+    <title>The HPCC Components in the values.yaml File</title>
+
+    <para>The HPCC Systems Helm charts all ship with stock/default values.
+    These delivered stock Helm chart has a set of default values ideally to be
+    used as a guide in configuring your deployment. Generally, every HPCC
+    Systems component is a list. That list defines the properties for each
+    component.</para>
+
+    <para>Previously we looked at the global, and the storage areas, with data
+    planes also being discussed in that section, then we looked at the
+    certificates, secrets, and security. This section will provide additional
+    details and any noteworthy insight for the HPCC Systems components defined
+    in the values.yaml file.</para>
+
+    <sect2 id="YAMLHPCC_Components">
+      <title>the HPCC Components</title>
+
+      <para>Some general concepts regarding the HPCC Systems components, as
+      the components are lists.If you had an [ ] empty value list, it would
+      invalidate that list elsewhere. The previous section detailed that
```

Comment:
I think there were some previous comments above that related to this (deleting contents of list to disable a component).
It would be good to consolidate the related comments together in 1 section.